### PR TITLE
Admin can edit subject after create course

### PR DESCRIPTION
**Pull request by NgocVu1994**

### CODE REVIEW

File: app/controllers/admin/course_subjects_controller.rb [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
+  def edit
+  end
+
+  def update
+    if @course_subject.update_attributes course_subject_params
+      flash[:success] = flash_message "updated"
+      redirect_to admin_course_course_subjects_path
+    else
+      flash[:failed] = flash_message "not_updated"
+      redirect_to :edit
```
Comment (thiennv57) [inline app/controllers/admin/course_subjects_controller.rb]: render chứ nhỉ??

**Comment by bapboy18** [comment]
ok :+1:

**Comment by vuducluan** [comment]
OK

**Comment by yongsokheng** [comment]
OK

**Pull request merged**